docs/datamap/preaggregate-datamap-guide.md

```diff
@@ -176,6 +176,9 @@ It will show all DataMaps created on main table.
     FROM sales
     GROUP BY country, sex
   ```
+  **NOTE**:
+   * Preaggregate datamap will be deprecated to later versions. It will be replaced by MV datamap.
```

Comment:
Please rephrase 
`Preaggregate datamap is deprecated and it is replaced by MV datamap.`